src/mme-app/utils/mmeCommonUtils.cpp

```diff
@@ -70,7 +71,9 @@ uint32_t MmeCommonUtils::allocateMtmsi()
 
 	while(1)
```

Comment:
Can we get rid of while, since under heavy load its likely to loop more number of times.